cli/tests/services.bats

```diff
@@ -1518,8 +1519,36 @@ EOF
 
   # Make sure services have stopped
   timeout 1s bash -c '
-    while "$FLOX_BIN" services status; do
+    both_services_stopped() {
+      one_status="$("$FLOX_BIN" services status --json one | jq -r -c .status)"
+      two_status="$("$FLOX_BIN" services status --json two | jq -r -c .status)"
+      [ "$one_status" = "Stopped" ] && [ "$two_status" = "Stopped" ]
+    }
+    
+    while ! both_services_stopped; do
       sleep .1
     done
   '
 }
+
+@test "status: prints requested services (before start)" {
+  setup_sleeping_services
+
+  run "$FLOX_BIN" services status --json one
+  assert_success
+
+  statuses="$(echo "$output" | jq --slurp -c 'map(.status)')"
+  assert_equal "$statuses" '["Stopped"]'
+}
```

Comment:
I think it's worth moving this next to the existing test (mentioned below) and asserting on the full output to make it clear that we omit the PID when services aren't started.

Reply:
Fixed